### ISSUE TITLE
Scratching bug

### ISSUE THREAD

**Issue by Ivan-iv510** [25huizengek1/ViTune#543]
### Steps to reproduce the bug

So when i listen to some music i hear scratching in the background (didnt have this problem on vimusic but it doesnt work anymore)

### Expected behavior

There shouldnt be that scratching 

### Actual behavior

There shouldnt be that scratching 

### ViTune version

V1.1.1

### What kind of build are you using?

Release (GitHub / F-Droid)

### Android version

Android 14

### Device info

Using s23 ultra

### Checklist

- [X] I am able to reach YouTube Music through the **official** app/website and can confirm that I live in a [supported country](https://support.google.com/youtubemusic/answer/6313540?hl=en) (and this country is not temporarily unavailable)
- [X] I am not using a proxy, VPN, device policy or anything else that restricts access to YouTube Music
- [X] This issue has not been submitted already (this is not a duplicate of another issue)

**Comment by Wqttzicue** [comment]
Uhh, i assume its something with your headphones or device. Anyways provide logs, cuz Ingek more likely cant reproduce that

**Comment by Ivan-iv510** [comment]
> Uhh, and I assume its something with your headphones or device. Anyways provide logs, cuz Ingek more likely cant reproduce that
> 
> Btw im using practically the same device as you, but i dont have that issue

Hello thank you i found out whats the issue, at least i think, i uninstalled and reinstalled the app and there was no bug, then i went to change "skip silence" and it appered again so i deinstalled and reinstalled again and now it works so i guess i cant change that setting.

**Comment by 25huizengek1** [comment]
Nope, I unfortunately cannot reproduce this. Please provide some logs with this.
